.env-template

```diff
@@ -45,14 +45,16 @@ YAGNA_DATADIR="."
 #ERC20_MAINNET_REQUIRED_CONFIRMATIONS=5
 
 ## ZkSync driver
+#ZKSYNC_RINKEBY_RPC_ADDRESS=https://rinkeby-api.zksync.io/jsrpc
+#ZKSYNC_MAINNET_RPC_ADDRESS=https://api.zksync.io/jsrpc
+#GLMSYNC_RINKEBY_RPC_ADDRESS=TODO: What here?
```

Comment:
Hmm? :thinking:

Reply:
```
## ZkSync driver
#ZKSYNC_RINKEBY_RPC_ADDRESS=https://rinkeby-api.zksync.io/jsrpc
#ZKSYNC_MAINNET_RPC_ADDRESS=https://api.zksync.io/jsrpc
#ZKSYNC_FAUCET_ADDR=

## GlmSync driver
#GLMSYNC_RINKEBY_RPC_ADDRESS=http://rinkeby-api.zksync.imapp.pl/jsrpc
#GLMSYNC_MAINNET_RPC_ADDRESS=https://api.zksync.imapp.pl/jsrpc
#GLMSYNC_FAUCET_ADDR=
```

Faucet addresses don't have default values because they're resolved from SRV records by default.